bench/bench.ml

```diff
@@ -46,6 +53,13 @@ let main () =
     Bench.Test.create ~name:"JSON writing assoc to channel" (fun () ->
       Out_channel.with_file "/dev/null" ~f:(fun oc ->
       ignore (Yojson.Safe.to_channel oc large_int_assoc)));
+    begin
+      let buf = Buffer.create 1000 in
+      Bench.Test.create ~name:"JSON seq roundtrip" (fun () ->
+        let stream = Yojson.Safe.seq_from_string ~buf streamable_string in
+        ignore (Yojson.Safe.seq_to_string ~buf stream)
+      )
+    end;
```

Comment:
Any particular reason for the `begin`/`end`?

Reply:
I need some bracketing for the `let buf = ... in ...`, otherwise it would consume following elements of the list I believe. I used begin..end, would you prefer parentheses?

Reply:
Ah, I looked at it and didn't expect the `;` to bind closer as a statement separator than as a list item separator but I guess thinking about it, it makes sense.